core/controllers/feedback.py

```diff
@@ -36,6 +36,12 @@ def get(self, exploration_id):
         self.values.update({
             'threads': [t.to_dict() for t in feedback_services.get_all_threads(
                 exploration_id, False)]})
+        if constants.USE_NEW_SUGGESTION_FRAMEWORK:
+            self.values.update({
+                'threads_with_suggestions': (
```

Comment:
Unclear from the name whether these are thread objects together with suggestion objects, or just thread objects (that correspond to suggestions). If the latter, then use the name `suggestion_thread_dicts` perhaps.

Reply:
Done